README.md

```diff
@@ -36,6 +36,9 @@ curl -LsSf https://astral.sh/uv/install.sh | sh
 
 # install Posting (will also quickly install Python 3.12 if needed)
 uv tool install --python 3.12 posting
+
+# run the TUI
+uv tool run --python 3.12 posting
```

Comment:
From my test, I could run without `uv`, just running `posting`, right after running `uv tool install --python 3.12 posting`

Isn't that your case?

Reply:
Yes, this is correct. After installing you should just be able to run `posting`.

Reply:
On this macOS machine I'm on python 3.10.9, which is likely the ground cause for that then.
Guessing that it wouldn’t be the same if I have >= 3.11 installed.

Not to fluent in python environments though, so this is how I got it running.

We should perhaps then just mention this `uv tool run --python 3.12 posting` as a secondary thing in the docs if `posting` isn’t working as it should. Until we find out what the real fix is.